account_banking_sepa_direct_debit/i18n/pt_BR.po

```diff
@@ -90,16 +90,12 @@ msgstr ""
 
 #. module: account_banking_sepa_direct_debit
 #: model:ir.ui.view,arch_db:account_banking_sepa_direct_debit.sepa_direct_debit_mandate_document
-msgid ""
-"<span class=\"col-xs-12 text-right\" style=\"font-size:8px;\">To be "
-"completed by the creditor</span>"
+msgid "To be completed by the creditor"
```

Comment:
This is done automatically by Transifex OCA bot, so no need to do it manually if you don't want.